src/clr-angular/data/datagrid/datagrid-row.html

```diff
@@ -3,10 +3,17 @@
   Clicking of that wrapper label will equate to clicking on the whole row, which triggers the checkbox to toggle.
 -->
 <label class="datagrid-row-clickable" *ngIf="selection.rowSelectionMode">
-  <ng-template [ngTemplateOutlet]="rowContent"></ng-template>
+  <clr-expandable-animation [clrExpandTrigger]="expanded" *ngIf="globalExpandable.hasExpandableRow">
```

Comment:
I am almost certain that it's breaking when the content is lazy loaded. The animation would run before the content. So we need to hook it with animate subject somehow.

Reply:
Yes, you are right, it's not hard to support this case. 
Done!